docs/plugin_development.rst

```diff
@@ -352,48 +427,57 @@ After ``cookicutter`` finishes its job, you'll get the following tree structure
+The ``spyder-pomodoro-timer`` folder has the same name you entered when you ran ``cookiecutter``.
+Inside it you will see a folder called ``spyder``, where we will place the code of our plugin.
 
 In the ``spyder`` directory you'll find the following files:
 
-* ``api.py``: where the functionality of the plugin is exposed to the rest of Spyder. That would allow additional functionality to be added from other plugins.
+* ``api.py``.
+  Exposes the functionality of the plugin to the rest of Spyder.
+  That allows adding additional functionality from other plugins.
 
-* ``plugin.py``: is the core of the plugin. Depending on the type of plugin we created, here you'll see an instance of ``SpyderDockablePlugin`` or ``SpyderPluginV2``.
+* ``plugin.py``.
+  This is the core of the plugin.
+  Depending on the type of plugin created, you will see an instance of either ``SpyderDockablePlugin`` or ``SpyderPluginV2``.
 
   * If it is a ``SpyderPluginV2`` you should set a constant class named ``CONTAINER_CLASS`` with an instance of ``PluginMainContainer``.
   * If it is a ``SpyderDockablePlugin`` you should set a constant class named ``WIDGET_CLASS`` with an instance of ``PluginMainWidget``.
 
-* ``container.py``: only used for ``SpyderPluginV2`` plugins. This file contains an instance of ``PluginMainContainer`` that holds a reference to all graphical elements (or widgets) that the plugin is going to add to the interface. This is necessary because Qt requires widgets to be children of other widgets before using them (otherwise they appear as floating windows). Since ``SpyderPluginV2`` is not a widget, we need a data structure (i.e. the container) that is a widget for that.
+* ``container.py``.
+  It is only used for ``SpyderPluginV2`` plugins.
+  This file contains an instance of ``PluginMainContainer``, which contains a reference to all graphical elements (or widgets) that the plugin will add to the interface.
+  This is necessary because Qt requires widgets to be children of other widgets before using them (otherwise they appear as floating windows).
+  Since ``SpyderPluginV2`` is not a widget, we need a data structure (i.e. the container) that is a widget for it.
 
-* ``widgets.py``: in this file we will add the graphical components of our plugin. If it is of type ``SpyderPluginV2`` and it does not have widgets, then it is not necessary.
-  We can also place here the instance of ``PluginMainWidget`` necessary for ``SpyderDockablePlugin``, if we are developing that kind of plugin.
+* ``widgets.py``.
+  Contains the graphical components of our plugin.
+  If the plugin is of type ``SpyderPluginV2`` and it does not have widgets, then it is not needed.
+  If the plugin is of type ``SpyderDockablePlugin``, this is where we can place the instance of ``PluginMainWidget`` that is needed for it.
```

Comment:
```suggestion
  For ``SpyderDockablePlugin`` or a ``SpyderPluginV2`` with widgets, the required instance of ``PluginMainWidget`` goes here.
  Not needed for a ``SpyderPluginV2`` without widgets.
```

Again, found this unnecessarily complex and somewhat confusing. I think I've interpreted it correctly here.

Reply:
Counterproposal, with some minor fixes:

```suggestion
  For a ``SpyderDockablePlugin`` or a ``SpyderPluginV2`` with widgets, the required instance of ``PluginMainWidget`` goes here.
  But it's not needed for a ``SpyderPluginV2`` without widgets.
```

Reply:
```suggestion
  This not needed for a ``SpyderPluginV2`` without widgets.
  For ``SpyderPluginV2`` with widgets or a ``SpyderDockablePlugin``, the required instance of ``PluginMainWidget`` goes here.
```

Fixed an incomplete sentence issue with the above and also re-ordered the sentences and clauses to be more clear and unambiguous.